source/sxl_traffic_light_controller.rst

```diff
@@ -1964,25 +1964,25 @@ at some point, but until then it might block requests in other direction.
    :label: M0022
    :caption: M0022
    :loc: H
-   :spec: >{\raggedright\arraybackslash}p{0.14\linewidth} p{0.20\linewidth} p{0.07\linewidth} p{0.15\linewidth} p{0.30\linewidth}
-
-   ============  =======  ==============================  ===============================================================
-   Name          Type     Value                           Comment
-   ============  =======  ==============================  ===============================================================
-   requestId     string   [id]                            A string that uniquely identifies the request on the controller
-   componentId   string   [id]                            ID of an RSMP component in the controller, typically a signal group
-   inputId       integer  [0-255]                         ID of an input, using the same numbering scheme as M0006
-   connectionId  integer  [0-255]                         ID of a connection, connecting an ingoing and an outgoing lane
-   approachId    integer  [0-16]                          ID of an intersection approach
-   laneInId      integer  [0-255]                         ID of an ingoing lane
-   laneOutId     integer  [0-255]                         ID of an outgoing lane
-   priorityId    integer  [0-255]                         ID of a priority
-   type          string   | -new                          | new: Initiate a new priority request
-                          | -update                       | update: Update an existing priority request
-                          | -cancel                       | cancel: Cancel an existing priority request
-   level         integer  [0-14]                          0: Lowest, 14: Highest
-   eta           integer  [0-255]                         (Optional) Estimated time of arrival to the intersection, in seconds
-   vehicleType   string   | -car                          (Optional) Vehicle type
+   :spec: >{\raggedright\arraybackslash}p{0.14\linewidth} p{0.07\linewidth} p{0.15\linewidth} p{0.30\linewidth}
+
+   ============  =======  ===================  ====================================================================
+   Name          Type     Value                Comment
```

Comment:
I removed the 'Command' column since we're not really used in any of the commands. But to be consistent maybe it should be in the the M0022 table, or removed from all the other commands. If we remove it from the tables, the command name must be documented somehwere else - unless we actually remove it from the commands but that's a bigger change.